lib/project_types/script/messages/messages.rb

```diff
@@ -76,6 +76,25 @@ module Messages
 
           packages_outdated_cause: "The following npm packages are out of date: %s.",
           packages_outdated_help: "Update them by running {{cyan:npm install --save-dev %s}}.",
+
+          invalid_build_script: "Invalid build script",
```

Comment:
I think these messages need a period at the end to be properly formatted in the error handler:
![image](https://user-images.githubusercontent.com/28009669/99842100-3bba5a00-2b3d-11eb-88c5-3699fb20ea05.png)

```suggestion
          invalid_build_script: "Invalid build script.",
```